docs/filter-jdbc_static.asciidoc

```diff
@@ -139,7 +150,9 @@ See <<plugins-{type}s-{plugin}-object_order>>.
 follow the positional ordering.
 <5> Specifies the event field that will store the looked-up data. If the lookup
 returns multiple columns, the data is stored as a JSON object within the field.
-<6> Takes data from the JSON object and stores it in top-level event fields for
+<6> Uses values for user fields if the user can not be found in the users database. 
+In which case the <<plugins-{type}s-{plugin}-tag_on_default_use>> tags list is used to mark the event.
```

Comment:
```suggestion
<6> Tags user fields with the format defined in <<plugins-{type}s-{plugin}-tag_on_default_use>> when a user is not found in the users database. 
```

Reply:
I think the meaning it's little bit different.
In case the user is not found in the database, then it still create a new event, but with the data that's provided into `default_hash`, which has some default values matching the columns.
So if a DB lookup result an empty set it brings the data from this "static default values" map, and tags the event with the list present in `tag_on_default_use`.

Reply:
Ha right. I should not have commited the proposal... :(

Reply:
I reverted my last commit

Reply:
Expect @karenzone come up with a nice and fluent sentence to express that concept and avoid the previous tongue twister